plugins/star-rating/frontend/public/templates/ratings-tab.html

```diff
@@ -0,0 +1,38 @@
+<div v-bind:class="[componentId]" class="ratings-tab-view">
+	<cly-header :title="i18n('feedback.ratings')">
+			<template v-slot:header-right></template>
+	</cly-header>
+  <div class="bu-level bu-m-5">
```

Comment:
instead of bu-m-5, i think you should wrap this entire component within cly-main, since it adds the global paddings and margins itself.

Reply:
so the idea is to wrap the main content of the view within cly-main component
Unless its headers or tabs.
header and tabs component stay outside the cly-main